packages/e2e-tests/plugins/iframed-multiple-stylesheets.php

```diff
@@ -7,13 +7,6 @@
  * @package gutenberg-test-iframed-multiple-stylesheets
  */
 
-add_action(
-	'setup_theme',
-	static function () {
-		add_theme_support( 'block-templates' );
-	}
-);
```

Comment:
The feature was used to simulate an iframed editor. This is no longer needed as the editor canvas renders in iframed by default now.